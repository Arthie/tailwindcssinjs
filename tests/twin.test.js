const pluginTester = require("babel-plugin-tester");
const plugin = require("babel-plugin-macros");

// const benchBuilder = require("./utils/benchBuilder");
// const test = benchBuilder("twin.macro");

const test = require("./utils/twinbench1");

pluginTester.default({
  pluginName: "twin",
  plugin,
  snapshot: true,
  babelOptions: {
    filename: __filename,
  },
  tests: test,
});
