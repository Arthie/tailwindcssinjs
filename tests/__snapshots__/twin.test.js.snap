// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`twin cursor-text                   : cursor-text                    1`] = `

import tw from 'twin.macro';
const css = tw\`cursor-text\`;

      ↓ ↓ ↓ ↓ ↓ ↓

const css = {
  cursor: "text",
};


`;

exports[`twin initial load (just import)    : initial load (just import)     1`] = `

import tw from 'twin.macro';

      ↓ ↓ ↓ ↓ ↓ ↓



`;

exports[`twin md:hover:cursor-text          : md:hover:cursor-text           1`] = `

import tw from 'twin.macro';
const css = tw\`md:hover:cursor-text\`;

      ↓ ↓ ↓ ↓ ↓ ↓

const css = {
  "@media (min-width: 768px)": {
    ":hover": {
      cursor: "text",
    },
  },
};


`;

exports[`twin md:hover:text-2xl             : md:hover:text-2xl              1`] = `

import tw from 'twin.macro';
const css = tw\`md:hover:text-2xl\`;

      ↓ ↓ ↓ ↓ ↓ ↓

const css = {
  "@media (min-width: 768px)": {
    ":hover": {
      fontSize: "1.5rem",
    },
  },
};


`;

exports[`twin md:hover:text-base            : md:hover:text-base             1`] = `

import tw from 'twin.macro';
const css = tw\`md:hover:text-base\`;

      ↓ ↓ ↓ ↓ ↓ ↓

const css = {
  "@media (min-width: 768px)": {
    ":hover": {
      fontSize: "1rem",
    },
  },
};


`;

exports[`twin md:hover:text-lg              : md:hover:text-lg               1`] = `

import tw from 'twin.macro';
const css = tw\`md:hover:text-lg\`;

      ↓ ↓ ↓ ↓ ↓ ↓

const css = {
  "@media (min-width: 768px)": {
    ":hover": {
      fontSize: "1.125rem",
    },
  },
};


`;

exports[`twin md:hover:text-sm              : md:hover:text-sm               1`] = `

import tw from 'twin.macro';
const css = tw\`md:hover:text-sm\`;

      ↓ ↓ ↓ ↓ ↓ ↓

const css = {
  "@media (min-width: 768px)": {
    ":hover": {
      fontSize: "0.875rem",
    },
  },
};


`;

exports[`twin md:hover:text-xl              : md:hover:text-xl               1`] = `

import tw from 'twin.macro';
const css = tw\`md:hover:text-xl\`;

      ↓ ↓ ↓ ↓ ↓ ↓

const css = {
  "@media (min-width: 768px)": {
    ":hover": {
      fontSize: "1.25rem",
    },
  },
};


`;

exports[`twin md:hover:text-xs              : md:hover:text-xs               1`] = `

import tw from 'twin.macro';
const css = tw\`md:hover:text-xs\`;

      ↓ ↓ ↓ ↓ ↓ ↓

const css = {
  "@media (min-width: 768px)": {
    ":hover": {
      fontSize: "0.75rem",
    },
  },
};


`;

exports[`twin multiple transforms 20 classes: multiple transforms 20 classes 1`] = `

import tw from 'twin.macro';
const css1 = tw\`cursor-text md:hover:cursor-text text-xs md:hover:text-xs text-sm md:hover:text-sm text-base md:hover:text-base text-lg md:hover:text-lg text-xl md:hover:text-xl text-2xl md:hover:text-2xl text-3xl md:hover:text-3xl text-4xl md:hover:text-4xl text-5xl md:hover:text-5xl\`;
const css2 = tw\`md:hover:text-sm text-base md:hover:text-base text-lg md:hover:text-lg text-xl md:hover:text-xl text-2xl md:hover:text-2xl text-3xl md:hover:text-3xl text-4xl md:hover:text-4xl text-5xl md:hover:text-5xl text-6xl md:hover:text-6xl text-left md:hover:text-left text-center\`;
const css3 = tw\`text-xl md:hover:text-xl text-2xl md:hover:text-2xl text-3xl md:hover:text-3xl text-4xl md:hover:text-4xl text-5xl md:hover:text-5xl text-6xl md:hover:text-6xl text-left md:hover:text-left text-center md:hover:text-center text-right md:hover:text-right text-justify md:hover:text-justify\`;
const css4 = tw\`md:hover:text-3xl text-4xl md:hover:text-4xl text-5xl md:hover:text-5xl text-6xl md:hover:text-6xl text-left md:hover:text-left text-center md:hover:text-center text-right md:hover:text-right text-justify md:hover:text-justify text-transparent md:hover:text-transparent text-current md:hover:text-current text-black\`;
const css5 = tw\`text-6xl md:hover:text-6xl text-left md:hover:text-left text-center md:hover:text-center text-right md:hover:text-right text-justify md:hover:text-justify text-transparent md:hover:text-transparent text-current md:hover:text-current text-black md:hover:text-black text-white md:hover:text-white text-gray-100 md:hover:text-gray-100\`;
const css6 = tw\`md:hover:text-center text-right md:hover:text-right text-justify md:hover:text-justify text-transparent md:hover:text-transparent text-current md:hover:text-current text-black md:hover:text-black text-white md:hover:text-white text-gray-100 md:hover:text-gray-100 text-gray-200 md:hover:text-gray-200 text-gray-300 md:hover:text-gray-300 text-gray-400\`;
const css7 = tw\`text-transparent md:hover:text-transparent text-current md:hover:text-current text-black md:hover:text-black text-white md:hover:text-white text-gray-100 md:hover:text-gray-100 text-gray-200 md:hover:text-gray-200 text-gray-300 md:hover:text-gray-300 text-gray-400 md:hover:text-gray-400 text-gray-500 md:hover:text-gray-500 text-gray-600 md:hover:text-gray-600\`;
const css8 = tw\`md:hover:text-black text-white md:hover:text-white text-gray-100 md:hover:text-gray-100 text-gray-200 md:hover:text-gray-200 text-gray-300 md:hover:text-gray-300 text-gray-400 md:hover:text-gray-400 text-gray-500 md:hover:text-gray-500 text-gray-600 md:hover:text-gray-600 text-gray-700 md:hover:text-gray-700 text-gray-800 md:hover:text-gray-800 text-gray-900\`;
const css9 = tw\`text-gray-200 md:hover:text-gray-200 text-gray-300 md:hover:text-gray-300 text-gray-400 md:hover:text-gray-400 text-gray-500 md:hover:text-gray-500 text-gray-600 md:hover:text-gray-600 text-gray-700 md:hover:text-gray-700 text-gray-800 md:hover:text-gray-800 text-gray-900 md:hover:text-gray-900 text-red-100 md:hover:text-red-100 text-red-200 md:hover:text-red-200\`;

      ↓ ↓ ↓ ↓ ↓ ↓

const css1 = {
  cursor: "text",
  fontSize: "3rem",
  "@media (min-width: 768px)": {
    ":hover": {
      cursor: "text",
      fontSize: "3rem",
    },
  },
};
const css2 = {
  fontSize: "4rem",
  textAlign: "center",
  "@media (min-width: 768px)": {
    ":hover": {
      fontSize: "4rem",
      textAlign: "left",
    },
  },
};
const css3 = {
  fontSize: "4rem",
  textAlign: "justify",
  "@media (min-width: 768px)": {
    ":hover": {
      fontSize: "4rem",
      textAlign: "justify",
    },
  },
};
const css4 = {
  fontSize: "4rem",
  textAlign: "justify",
  color: "rgba(0, 0, 0, var(--text-opacity))",
  "--text-opacity": "1",
  "@media (min-width: 768px)": {
    ":hover": {
      fontSize: "4rem",
      textAlign: "justify",
      color: "currentColor",
    },
  },
};
const css5 = {
  fontSize: "4rem",
  textAlign: "justify",
  color: "rgba(247, 250, 252, var(--text-opacity))",
  "--text-opacity": "1",
  "@media (min-width: 768px)": {
    ":hover": {
      fontSize: "4rem",
      textAlign: "justify",
      color: "rgba(247, 250, 252, var(--text-opacity))",
      "--text-opacity": "1",
    },
  },
};
const css6 = {
  textAlign: "justify",
  color: "rgba(203, 213, 224, var(--text-opacity))",
  "--text-opacity": "1",
  "@media (min-width: 768px)": {
    ":hover": {
      textAlign: "justify",
      color: "rgba(226, 232, 240, var(--text-opacity))",
      "--text-opacity": "1",
    },
  },
};
const css7 = {
  color: "rgba(113, 128, 150, var(--text-opacity))",
  "--text-opacity": "1",
  "@media (min-width: 768px)": {
    ":hover": {
      color: "rgba(113, 128, 150, var(--text-opacity))",
      "--text-opacity": "1",
    },
  },
};
const css8 = {
  "--text-opacity": "1",
  color: "rgba(26, 32, 44, var(--text-opacity))",
  "@media (min-width: 768px)": {
    ":hover": {
      "--text-opacity": "1",
      color: "rgba(45, 55, 72, var(--text-opacity))",
    },
  },
};
const css9 = {
  "--text-opacity": "1",
  color: "rgba(254, 215, 215, var(--text-opacity))",
  "@media (min-width: 768px)": {
    ":hover": {
      "--text-opacity": "1",
      color: "rgba(254, 215, 215, var(--text-opacity))",
    },
  },
};


`;

exports[`twin test 10 classes               : test 10 classes                1`] = `

import tw from 'twin.macro';
const css = tw\`cursor-text md:hover:cursor-text text-xs md:hover:text-xs text-sm md:hover:text-sm text-base md:hover:text-base text-lg md:hover:text-lg\`;

      ↓ ↓ ↓ ↓ ↓ ↓

const css = {
  cursor: "text",
  fontSize: "1.125rem",
  "@media (min-width: 768px)": {
    ":hover": {
      cursor: "text",
      fontSize: "1.125rem",
    },
  },
};


`;

exports[`twin test 20 classes               : test 20 classes                1`] = `

import tw from 'twin.macro';
const css = tw\`cursor-text md:hover:cursor-text text-xs md:hover:text-xs text-sm md:hover:text-sm text-base md:hover:text-base text-lg md:hover:text-lg text-xl md:hover:text-xl text-2xl md:hover:text-2xl text-3xl md:hover:text-3xl text-4xl md:hover:text-4xl text-5xl md:hover:text-5xl\`;

      ↓ ↓ ↓ ↓ ↓ ↓

const css = {
  cursor: "text",
  fontSize: "3rem",
  "@media (min-width: 768px)": {
    ":hover": {
      cursor: "text",
      fontSize: "3rem",
    },
  },
};


`;

exports[`twin test 40 classes               : test 40 classes                1`] = `

import tw from 'twin.macro';
const css = tw\`cursor-text md:hover:cursor-text text-xs md:hover:text-xs text-sm md:hover:text-sm text-base md:hover:text-base text-lg md:hover:text-lg text-xl md:hover:text-xl text-2xl md:hover:text-2xl text-3xl md:hover:text-3xl text-4xl md:hover:text-4xl text-5xl md:hover:text-5xl text-6xl md:hover:text-6xl text-left md:hover:text-left text-center md:hover:text-center text-right md:hover:text-right text-justify md:hover:text-justify text-transparent md:hover:text-transparent text-current md:hover:text-current text-black md:hover:text-black text-white md:hover:text-white text-gray-100 md:hover:text-gray-100\`;

      ↓ ↓ ↓ ↓ ↓ ↓

const css = {
  cursor: "text",
  fontSize: "4rem",
  textAlign: "justify",
  color: "rgba(247, 250, 252, var(--text-opacity))",
  "--text-opacity": "1",
  "@media (min-width: 768px)": {
    ":hover": {
      fontSize: "3rem",
      textAlign: "justify",
      color: "rgba(247, 250, 252, var(--text-opacity))",
      "--text-opacity": "1",
      cursor: "text",
    },
  },
};


`;

exports[`twin test 80 classes               : test 80 classes                1`] = `

import tw from 'twin.macro';
const css = tw\`cursor-text md:hover:cursor-text text-xs md:hover:text-xs text-sm md:hover:text-sm text-base md:hover:text-base text-lg md:hover:text-lg text-xl md:hover:text-xl text-2xl md:hover:text-2xl text-3xl md:hover:text-3xl text-4xl md:hover:text-4xl text-5xl md:hover:text-5xl text-6xl md:hover:text-6xl text-left md:hover:text-left text-center md:hover:text-center text-right md:hover:text-right text-justify md:hover:text-justify text-transparent md:hover:text-transparent text-current md:hover:text-current text-black md:hover:text-black text-white md:hover:text-white text-gray-100 md:hover:text-gray-100 text-gray-200 md:hover:text-gray-200 text-gray-300 md:hover:text-gray-300 text-gray-400 md:hover:text-gray-400 text-gray-500 md:hover:text-gray-500 text-gray-600 md:hover:text-gray-600 text-gray-700 md:hover:text-gray-700 text-gray-800 md:hover:text-gray-800 text-gray-900 md:hover:text-gray-900 text-red-100 md:hover:text-red-100 text-red-200 md:hover:text-red-200 text-red-300 md:hover:text-red-300 text-red-400 md:hover:text-red-400 text-red-500 md:hover:text-red-500 text-red-600 md:hover:text-red-600 text-red-700 md:hover:text-red-700 text-red-800 md:hover:text-red-800 text-red-900 md:hover:text-red-900 text-orange-100 md:hover:text-orange-100 text-orange-200 md:hover:text-orange-200 text-orange-300 md:hover:text-orange-300\`;

      ↓ ↓ ↓ ↓ ↓ ↓

const css = {
  cursor: "text",
  fontSize: "4rem",
  textAlign: "justify",
  color: "rgba(251, 211, 141, var(--text-opacity))",
  "--text-opacity": "1",
  "@media (min-width: 768px)": {
    ":hover": {
      "--text-opacity": "1",
      color: "rgba(247, 250, 252, var(--text-opacity))",
      fontSize: "3rem",
      textAlign: "justify",
      cursor: "text",
    },
  },
};


`;

exports[`twin test 160 classes              : test 160 classes               1`] = `

import tw from 'twin.macro';
const css = tw\`cursor-text md:hover:cursor-text text-xs md:hover:text-xs text-sm md:hover:text-sm text-base md:hover:text-base text-lg md:hover:text-lg text-xl md:hover:text-xl text-2xl md:hover:text-2xl text-3xl md:hover:text-3xl text-4xl md:hover:text-4xl text-5xl md:hover:text-5xl text-6xl md:hover:text-6xl text-left md:hover:text-left text-center md:hover:text-center text-right md:hover:text-right text-justify md:hover:text-justify text-transparent md:hover:text-transparent text-current md:hover:text-current text-black md:hover:text-black text-white md:hover:text-white text-gray-100 md:hover:text-gray-100 text-gray-200 md:hover:text-gray-200 text-gray-300 md:hover:text-gray-300 text-gray-400 md:hover:text-gray-400 text-gray-500 md:hover:text-gray-500 text-gray-600 md:hover:text-gray-600 text-gray-700 md:hover:text-gray-700 text-gray-800 md:hover:text-gray-800 text-gray-900 md:hover:text-gray-900 text-red-100 md:hover:text-red-100 text-red-200 md:hover:text-red-200 text-red-300 md:hover:text-red-300 text-red-400 md:hover:text-red-400 text-red-500 md:hover:text-red-500 text-red-600 md:hover:text-red-600 text-red-700 md:hover:text-red-700 text-red-800 md:hover:text-red-800 text-red-900 md:hover:text-red-900 text-orange-100 md:hover:text-orange-100 text-orange-200 md:hover:text-orange-200 text-orange-300 md:hover:text-orange-300 text-orange-400 md:hover:text-orange-400 text-orange-500 md:hover:text-orange-500 text-orange-600 md:hover:text-orange-600 text-orange-700 md:hover:text-orange-700 text-orange-800 md:hover:text-orange-800 text-orange-900 md:hover:text-orange-900 text-yellow-100 md:hover:text-yellow-100 text-yellow-200 md:hover:text-yellow-200 text-yellow-300 md:hover:text-yellow-300 text-yellow-400 md:hover:text-yellow-400 text-yellow-500 md:hover:text-yellow-500 text-yellow-600 md:hover:text-yellow-600 text-yellow-700 md:hover:text-yellow-700 text-yellow-800 md:hover:text-yellow-800 text-yellow-900 md:hover:text-yellow-900 text-green-100 md:hover:text-green-100 text-green-200 md:hover:text-green-200 text-green-300 md:hover:text-green-300 text-green-400 md:hover:text-green-400 text-green-500 md:hover:text-green-500 text-green-600 md:hover:text-green-600 text-green-700 md:hover:text-green-700 text-green-800 md:hover:text-green-800 text-green-900 md:hover:text-green-900 text-teal-100 md:hover:text-teal-100 text-teal-200 md:hover:text-teal-200 text-teal-300 md:hover:text-teal-300 text-teal-400 md:hover:text-teal-400 text-teal-500 md:hover:text-teal-500 text-teal-600 md:hover:text-teal-600 text-teal-700 md:hover:text-teal-700 text-teal-800 md:hover:text-teal-800 text-teal-900 md:hover:text-teal-900 text-blue-100 md:hover:text-blue-100 text-blue-200 md:hover:text-blue-200 text-blue-300 md:hover:text-blue-300 text-blue-400 md:hover:text-blue-400 text-blue-500 md:hover:text-blue-500 text-blue-600 md:hover:text-blue-600 text-blue-700 md:hover:text-blue-700\`;

      ↓ ↓ ↓ ↓ ↓ ↓

const css = {
  cursor: "text",
  fontSize: "4rem",
  textAlign: "justify",
  color: "rgba(43, 108, 176, var(--text-opacity))",
  "--text-opacity": "1",
  "@media (min-width: 768px)": {
    ":hover": {
      "--text-opacity": "1",
      color: "rgba(247, 250, 252, var(--text-opacity))",
      fontSize: "3rem",
      textAlign: "justify",
      cursor: "text",
    },
  },
};


`;

exports[`twin test 220 classes              : test 220 classes               1`] = `

import tw from 'twin.macro';
const css = tw\`cursor-text md:hover:cursor-text text-xs md:hover:text-xs text-sm md:hover:text-sm text-base md:hover:text-base text-lg md:hover:text-lg text-xl md:hover:text-xl text-2xl md:hover:text-2xl text-3xl md:hover:text-3xl text-4xl md:hover:text-4xl text-5xl md:hover:text-5xl text-6xl md:hover:text-6xl text-left md:hover:text-left text-center md:hover:text-center text-right md:hover:text-right text-justify md:hover:text-justify text-transparent md:hover:text-transparent text-current md:hover:text-current text-black md:hover:text-black text-white md:hover:text-white text-gray-100 md:hover:text-gray-100 text-gray-200 md:hover:text-gray-200 text-gray-300 md:hover:text-gray-300 text-gray-400 md:hover:text-gray-400 text-gray-500 md:hover:text-gray-500 text-gray-600 md:hover:text-gray-600 text-gray-700 md:hover:text-gray-700 text-gray-800 md:hover:text-gray-800 text-gray-900 md:hover:text-gray-900 text-red-100 md:hover:text-red-100 text-red-200 md:hover:text-red-200 text-red-300 md:hover:text-red-300 text-red-400 md:hover:text-red-400 text-red-500 md:hover:text-red-500 text-red-600 md:hover:text-red-600 text-red-700 md:hover:text-red-700 text-red-800 md:hover:text-red-800 text-red-900 md:hover:text-red-900 text-orange-100 md:hover:text-orange-100 text-orange-200 md:hover:text-orange-200 text-orange-300 md:hover:text-orange-300 text-orange-400 md:hover:text-orange-400 text-orange-500 md:hover:text-orange-500 text-orange-600 md:hover:text-orange-600 text-orange-700 md:hover:text-orange-700 text-orange-800 md:hover:text-orange-800 text-orange-900 md:hover:text-orange-900 text-yellow-100 md:hover:text-yellow-100 text-yellow-200 md:hover:text-yellow-200 text-yellow-300 md:hover:text-yellow-300 text-yellow-400 md:hover:text-yellow-400 text-yellow-500 md:hover:text-yellow-500 text-yellow-600 md:hover:text-yellow-600 text-yellow-700 md:hover:text-yellow-700 text-yellow-800 md:hover:text-yellow-800 text-yellow-900 md:hover:text-yellow-900 text-green-100 md:hover:text-green-100 text-green-200 md:hover:text-green-200 text-green-300 md:hover:text-green-300 text-green-400 md:hover:text-green-400 text-green-500 md:hover:text-green-500 text-green-600 md:hover:text-green-600 text-green-700 md:hover:text-green-700 text-green-800 md:hover:text-green-800 text-green-900 md:hover:text-green-900 text-teal-100 md:hover:text-teal-100 text-teal-200 md:hover:text-teal-200 text-teal-300 md:hover:text-teal-300 text-teal-400 md:hover:text-teal-400 text-teal-500 md:hover:text-teal-500 text-teal-600 md:hover:text-teal-600 text-teal-700 md:hover:text-teal-700 text-teal-800 md:hover:text-teal-800 text-teal-900 md:hover:text-teal-900 text-blue-100 md:hover:text-blue-100 text-blue-200 md:hover:text-blue-200 text-blue-300 md:hover:text-blue-300 text-blue-400 md:hover:text-blue-400 text-blue-500 md:hover:text-blue-500 text-blue-600 md:hover:text-blue-600 text-blue-700 md:hover:text-blue-700 text-blue-800 md:hover:text-blue-800 text-blue-900 md:hover:text-blue-900 text-indigo-100 md:hover:text-indigo-100 text-indigo-200 md:hover:text-indigo-200 text-indigo-300 md:hover:text-indigo-300 text-indigo-400 md:hover:text-indigo-400 text-indigo-500 md:hover:text-indigo-500 text-indigo-600 md:hover:text-indigo-600 text-indigo-700 md:hover:text-indigo-700 text-indigo-800 md:hover:text-indigo-800 text-indigo-900 md:hover:text-indigo-900 text-purple-100 md:hover:text-purple-100 text-purple-200 md:hover:text-purple-200 text-purple-300 md:hover:text-purple-300 text-purple-400 md:hover:text-purple-400 text-purple-500 md:hover:text-purple-500 text-purple-600 md:hover:text-purple-600 text-purple-700 md:hover:text-purple-700 text-purple-800 md:hover:text-purple-800 text-purple-900 md:hover:text-purple-900 text-pink-100 md:hover:text-pink-100 text-pink-200 md:hover:text-pink-200 text-pink-300 md:hover:text-pink-300 text-pink-400 md:hover:text-pink-400 text-pink-500 md:hover:text-pink-500 text-pink-600 md:hover:text-pink-600 text-pink-700 md:hover:text-pink-700 text-pink-800 md:hover:text-pink-800 text-pink-900 md:hover:text-pink-900 select-text md:hover:select-text\`;

      ↓ ↓ ↓ ↓ ↓ ↓

const css = {
  cursor: "text",
  fontSize: "4rem",
  textAlign: "justify",
  color: "rgba(112, 36, 89, var(--text-opacity))",
  "--text-opacity": "1",
  userSelect: "text",
  "@media (min-width: 768px)": {
    ":hover": {
      "--text-opacity": "1",
      color: "rgba(26, 32, 44, var(--text-opacity))",
      userSelect: "text",
      textAlign: "right",
      cursor: "text",
      fontSize: "4rem",
    },
  },
};


`;

exports[`twin text-2xl                      : text-2xl                       1`] = `

import tw from 'twin.macro';
const css = tw\`text-2xl\`;

      ↓ ↓ ↓ ↓ ↓ ↓

const css = {
  fontSize: "1.5rem",
};


`;

exports[`twin text-3xl                      : text-3xl                       1`] = `

import tw from 'twin.macro';
const css = tw\`text-3xl\`;

      ↓ ↓ ↓ ↓ ↓ ↓

const css = {
  fontSize: "1.875rem",
};


`;

exports[`twin text-base                     : text-base                      1`] = `

import tw from 'twin.macro';
const css = tw\`text-base\`;

      ↓ ↓ ↓ ↓ ↓ ↓

const css = {
  fontSize: "1rem",
};


`;

exports[`twin text-lg                       : text-lg                        1`] = `

import tw from 'twin.macro';
const css = tw\`text-lg\`;

      ↓ ↓ ↓ ↓ ↓ ↓

const css = {
  fontSize: "1.125rem",
};


`;

exports[`twin text-sm                       : text-sm                        1`] = `

import tw from 'twin.macro';
const css = tw\`text-sm\`;

      ↓ ↓ ↓ ↓ ↓ ↓

const css = {
  fontSize: "0.875rem",
};


`;

exports[`twin text-xl                       : text-xl                        1`] = `

import tw from 'twin.macro';
const css = tw\`text-xl\`;

      ↓ ↓ ↓ ↓ ↓ ↓

const css = {
  fontSize: "1.25rem",
};


`;

exports[`twin text-xs                       : text-xs                        1`] = `

import tw from 'twin.macro';
const css = tw\`text-xs\`;

      ↓ ↓ ↓ ↓ ↓ ↓

const css = {
  fontSize: "0.75rem",
};


`;
